README.md

```diff
@@ -417,11 +437,15 @@ Exceeding rate limit:
 ```
 
 ## CORS
-Support Cross Origin Resource Sharing (CORS) for AJAX requests. You can read the CORS W3C working draft, or this intro from the HTML 5 Security Guide.
+Support Cross Origin Resource Sharing (CORS) for AJAX requests.
+
+Resources:
+- [CORS W3C working draft](https://www.w3.org/TR/cors/)
+- [HTML5 Rocks](http://www.html5rocks.com/en/tutorials/cors/).
```

Comment:
couldn't find the HTML 5 Sec guide link, so tentatively replaced it with this link. Could remove it from the list though. Or omit a resources list at all.